inference-engine/tests/functional/inference_engine/CMakeLists.txt

```diff
@@ -14,13 +14,16 @@ set(LINK_LIBRARIES
     inference_engine_transformations
     openvino::itt
     openvino::conditional_compilation
+    sharedTestClasses
 )
 set(DEPENDENCIES
     mock_engine
     inference_engine_ir_reader
     inference_engine_ir_v7_reader
     template_extension
+    funcTestUtils
```

Comment:
Do we still need `funcTestUtils` for serialization SLT?

Reply:
Done